google-cloud-datastore/samples/README.md

```diff
@@ -0,0 +1,60 @@
+# Cloud Datastore Sample
+
+This sample provides example code for
+[cloud.google.com/datastore/docs](https://cloud.google.com/datastore/docs).
+
+## Setup
+
+Before you can run or test the sample, you will need to enable the Cloud Datastore API in the [Google Developers Console](https://console.developers.google.com/projectselector/apis/api/datastore/overview).
+
+## Testing
+
+The tests for the sample are integration tests that run against the Datastore
+service and require authentication.
+
+### Authenticating
+
+Set one of the following environment variables to your Google Cloud Platform
+project ID:
+
+* `DATASTORE_DATASET`
+* `DATASTORE_PROJECT`
+* `GCLOUD_PROJECT`
```

Comment:
Don't list the `GCLOUD_*` environment variables, as we consider those deprecated.

Reply:
Done.